database_admin/migrations/045_bulk_delete.down.sql

```diff
@@ -0,0 +1,19 @@
+CREATE OR REPLACE FUNCTION delete_culled_systems(delete_limit INTEGER)
```

Comment:
ok, so it should be possible to revert 045->044, right? Great. And as there are no schema updates 45->46 so it should be also possible to revert 46->44.

Reply:
Yeah, copied the old variant of the function into the down migration.